crates/ty_python_semantic/resources/mdtest/replace.md

```diff
@@ -0,0 +1,64 @@
+# Replace
+
+The replace function and protocol added in Python 3.13:
+<https://docs.python.org/3/whatsnew/3.13.html#copy>
+
+## `replace()` function
+
+It is present in the `copy` module.
+
+```toml
+[environment]
+python-version = "3.13"
+```
```

Comment:
If you move this block up to the top level `# Replace` section, it will be active for all sub-sections. I think it makes sense to do this here, as every test will otherwise need to have one of these.